1Password/1Password.pkg.recipe

```diff
@@ -41,11 +41,11 @@
                 <key>app_path</key>
                 <string>%input_path%</string>
                 <key>pkg_path</key>
-                <string>%RECIPE_CACHE_DIR%/%NAME%-%DOWNLOAD_ARCH%-%version%.pkg</string>
+                <string>%RECIPE_CACHE_DIR%/%NAME%-%DOWNLOAD_ARCH%-%version%-%DOWNLOAD_BUILD%.pkg</string>
```

Comment:
Thanks, this is a better way to construct the download URL. Minor nit - my preference is for version to always come at the end. 

```suggestion
                <string>%RECIPE_CACHE_DIR%/%NAME%-%DOWNLOAD_BUILD%-%DOWNLOAD_ARCH%-%version%.pkg</string>
```

Reply:
Works for me!